integration-tests/src/test/java/org/apache/fineract/integrationtests/LoanDisbursementDetailsIntegrationTest.java

```diff
@@ -347,6 +350,83 @@ public void createApproveAndValidateMultiDisburseLoan() throws ParseException {
         this.editLoanDisbursementDetails();
     }
 
+    @Test
```

Comment:
Missing test case:
1. There are multiple disbursement
1.1 Undo the 2nd one, check whether it is remains active, the principal amount is correct, the scheduling is correct